src/Components/TextImageEdit/textImageEdit.jsx

```diff
@@ -173,29 +185,38 @@ export const TextImageEdit = ({ prop }) => {
       {disabledState && (
         <div className="flex-col flex-center">
           <div className="flex-row features-wrapper">
-            {likeCount > 0 ? (
-              <h1
-                onClick={() => dispatch(dislikePost(_id))}
-                className="text-center"
-              >
-                <AiFillHeart className="fill-heart" />
-                {`${likeCount}Likes`}
-              </h1>
-            ) : (
-              <h1 onClick={() => dispatch(likePost(_id))}>
-                <AiOutlineHeart /> {`${likeCount}Likes`}
+            {location.pathname !== "/bookmark" &&
+              (likeCount > 0 ? (
+                <h1
+                  onClick={() => dispatch(dislikePost({ postId: _id, token }))}
+                  className="text-center"
+                >
+                  <AiFillHeart className="fill-heart" />
+                  {`${likeCount}Likes`}
+                </h1>
+              ) : (
+                <h1 onClick={() => dispatch(likePost({ postId: _id, token }))}>
+                  <AiOutlineHeart /> {`${likeCount}Likes`}
+                </h1>
+              ))}
+            {location.pathname !== "/bookmark" && (
+              <h1 onClick={() => setShowComment((state) => !state)}>
+                {" "}
+                <FaRegComment />{" "}
               </h1>
             )}
-            <h1 onClick={() => setShowComment((state) => !state)}>
-              {" "}
-              <FaRegComment />{" "}
-            </h1>
             {BookMarkHandler(_id) ? (
-              <h1 onClick={() => dispatch(removeBookMarkPost(_id))}>
+              <h1

```

Comment:
You can name function using camelCase. while reading code it creates confusion that whether you are using any component or function